### PR TITLE
Compatibility with new sf toolbar/profiler design

### PR DESCRIPTION
**Pull request by norkunas**

### CODE REVIEW

**Comment by JulienItard** [comment]
merge would be great !

File: Resources/views/Collector/geocoder.html.twig [hunk 1]
```diff
@@ -1,69 +1,126 @@
...
+            <div class="sf-toolbar-info-piece">
+                <b>Total time</b>
+                <span>{{ '%0.2f'|format(collector.time) }} ms</span>
+            </div>
+        {% endset %}
+
+        {% include 'WebProfilerBundle:Profiler:toolbar_item.html.twig' with { 'link': profiler_url } %}
+    {% elseif collector.requestsCount %}
+        {% set icon %}
+            {{ include('@BazingaGeocoder/Collector/icon.svg') }}
```
Comment (hason) [inline Resources/views/Collector/geocoder.html.twig]: The file `icon.svg` is missing.

**Comment by willdurand** [comment]
Hi. Thanks for this!

Two questions:

1. Can anyone confirm that it works well?
2. Should we bump Symfony minimum version to something else?

**Comment by willdurand** [comment]
cc @geocoder-php/owners

**Comment by willdurand** [comment]
cc @geocoder-php/geocoder

File: Resources/views/Collector/geocoder.html.twig [hunk 2]
```diff
@@ -1,69 +1,126 @@
...
 
 {% block panel %}
+    {% set profiler_markup_version = profiler_markup_version|default(1) %}
     <h2>Geocoder Requests</h2>
 
     {% if not collector.requestsCount %}
-        <p><em>No requests.</em></p>
-    {% else %}
+        <div class="empty">
+            <p>No requests..</p>
```
Comment (Baachi) [inline Resources/views/Collector/geocoder.html.twig]: Typo

File: Resources/views/Collector/icon.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+    <path fill="#AAAAAA" d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"/>
+    <path d="M0 0h24v24H0z" fill="none"/>
```
Comment (Sander-Toonen) [inline Resources/views/Collector/icon.svg]: Remove the path "M0 0h24v24H0z", it overlaps the icon.

**Comment by Sander-Toonen** [comment]
Looks good to me (Tested on SF3.1). I haven't tested Ajax calls though.

**Comment by norkunas** [comment]
Almost year has passed, will this be merged?

**Comment by Baachi** [comment]
For me it looks great. :+1: 

The tests failures aren't realted to this PR, right?

**Comment by Sander-Toonen** [comment]
@Baachi No, some package couldn't be fetched at the time of the tests. Rerunning the tests wouldn't hurt though.

**Comment by Nyholm** [comment]
Sorry for the late reponse. Could you rebase your PR and I'll be happy to review and merge it.

**Comment by Nyholm** [comment]
Thank you. This is fixed in master.

**Comment by Sander-Toonen** [comment]
Looking goood! Thank you!

**Pull request closed without merging**